packages/ckeditor5-table/docs/_snippets/features/table-default-properties.html

```diff
@@ -1,8 +1,8 @@
 <style id="table-default-properties-styles">
     .live-snippet .table-default-properties-container .ck.ck-content .table {
-        float: right;
-        width: 500px;
-        height: 250px;
+        float: left;
+        width: 550px;
+        height: 450px;
     }
```

Comment:
This change must be synchronized with the editor's configuration. See https://github.com/ckeditor/ckeditor5/blob/4cc1422c32dfb6a3ef2e1da0db0752c4795c96c5/packages/ckeditor5-table/docs/_snippets/features/table-default-properties.js#L14-L33

Reply:
Addressed in the commit.